SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -574,28 +546,28 @@ Buffer {
 	}
 
 	copyData { arg buf, dstStartAt = 0, srcStartAt = 0, numSamples = -1;
-		if(bufnum.isNil) { Error("Cannot call % on a % that has been freed".format(thisMethod.name, this.class.name)).throw };
-		server.listSendMsg([\b_gen, buf.bufnum, "copy", dstStartAt, bufnum, srcStartAt, numSamples])
+		// server.listSendMsg([\b_gen, buf.bufnum, "copy", dstStartAt, bufnum, srcStartAt, numSamples])
```

Comment:
remove?

Reply:
Yeah, OK.

Reply:
Actually no need: https://github.com/supercollider/supercollider/blob/8e3a8a9b2d90b1f4c04b8c06ca8e500d9d13be41/SCClassLibrary/Common/Control/Buffer.sc#L548-L556

There is no stray comment here... don't know why it's in the diff.

Reply:
Oh, I see it now. Male pattern blindness.

Removed.